### DISCUSSION TITLE
How to display filebrowser version in frontend

### DISCUSSION THREAD

**Discussion by mj321** [filebrowser/filebrowser#2841] [General]
The filebrowser demo at demo.filebrowser.org shows the current filebrowser version number in the sidebar.

I built filebrowser from source, for me it only says (untracked).

I searched the whole codebase and couldn't find the current version anywhere, so what's the trick here?

**Comment by Nerahikada** [comment]
I had the same problem, but when I created a release, it seemed to show the version. I found out that goreleaser seems to be doing this work:
https://github.com/filebrowser/filebrowser/blob/fe5ca74aa1e4257e5cb36f1de58daa0c3548319f/.goreleaser.yml#L10

**Reply by Nerahikada** [reply]
You can edit `version.go` to show any version you want:
https://github.com/filebrowser/filebrowser/blob/master/version/version.go

**Reply by mj321** [reply]
Thanks for your response, but i'm still confused that  the current version is not in the code base in any shape or form.
